frontend/src/components/DicomViewer/DicomImageRenderer.js

```diff
@@ -29,6 +32,24 @@ const DicomImageRenderer = (props) => {
   );
   const [rendererData, setRendererData] = useState();
 
+  const [mouseCoordinates, setMouseCoordinates] = useState({ x: 0, y: 0, value: 0 });
+
+  const [newCameraPosition, setOriginalCameraPosition] = useState();
```

Comment:
These variables should be called originalCameraPosition and setOriginalCameraPosition otherwise the names are a bit misleading